Charts/glazed/templates/configmap.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-configmap
```

```suggestion
  name: {{ include "helm.fullname" . }}-configmap
```

fullname handles the case where the Release contains multiple Charts